opencti-platform/opencti-graphql/src/manager/expiredManager.js

```diff
@@ -94,7 +94,7 @@ const expireHandler = async () => {
 const initExpiredManager = () => {
   let scheduler;
   return {
-    start: () => {
+    start: async () => {
```

Comment:
this change seems unrelated to error/log management, is it intended ?

Reply:
Its unrelated, just a warning fix